### ISSUE TITLE
Feature: Third option for confirm prompt

### ISSUE THREAD

**Issue by emmalexandria** [console-rs/dialoguer#307]
It's very common to have a confirm prompt with three options, being Yes, No, and All. All being an option to skip further confirmation prompts of the same kind (e.g when doing something to large number of files). Something similar would be very useful to have in Dialoguer, if it's reasonably possible within the codebase.

**Comment by agingorange** [comment]
Yes, or for a menu system responding to one letter where you choose an option, without having to press Enter to move through the menu tree.

**Comment by VorpalBlade** [comment]
I was looking for a "Yes/No/show Diff/interactive Shell" personally. So a MultiConfirm or SingleLineMenu or some such would be much appreciated
